src/rvsdg/egglog_optimizer/fast_analyses.egg

```diff
@@ -0,0 +1,24 @@
+; Note that because the merge function is set-union 
```

Comment:
I think splitting this code into `.egg` and `.rs` is confusing- I would put all of it in `.rs`